include/alpaka/atomic/AtomicCpu.hpp

```diff
@@ -6,10 +6,11 @@
 
 #include "alpaka/core/BoostPredef.hpp"
 
-// clang 9 fails at compile time when using boost::atomic_ref
+// clang 9/10/11 together with nvcc<11.6.0 as host compiler fails at compile time when using boost::atomic_ref
 #ifdef BOOST_COMP_CLANG_AVAILABLE
-#    if BOOST_COMP_CLANG < BOOST_VERSION_NUMBER(11, 0, 0)
-#        ifndef ALPAKA_DISABLE_ATOMIC_ATOMICREF
+#    if(BOOST_COMP_CLANG < BOOST_VERSION_NUMBER(12, 0, 0) && BOOST_COMP_NVCC                                          \
+        && BOOST_COMP_NVCC < BOOST_VERSION_NUMBER(11, 6, 0))
+#        if !defined(ALPAKA_DISABLE_ATOMIC_ATOMICREF)
```

Comment:
do we still need this with the `alpaka::` change ?

Reply:
yes, it is a new window bug :-(